c2corg_ui/templates/helpers.html

```diff
@@ -61,37 +61,6 @@
   </ul>
```

Comment:
This file ``c2corg_ui/templates/helpers.html`` should be removed since splitted in several ``c2corg_ui/templates/helpers/*.html`` files.